src/components/spaces/show/Tutorial/pageStyle.css

```diff
@@ -0,0 +1,66 @@
+@import './styles/variables.css';
+
+.tutorialPage {
+  margin-bottom: 2em;
+  min-height: 35em;
+
+  .header {
+    margin-bottom: 1em;
+    text-align: center;
+  }
+
+  h3 {
+    text-align: center;
+    color: #666;
+    margin: 0.8em auto;
+    font-size: 1.4em;
+  }
+
+  p {
+    font-size: 1.1em;
+    text-align: justify;
+  }
+
+  .image {
+    width: 100%;
+    background-color: #dfe1e4;
+    border-radius: 3px;
+    margin-bottom: 2em;
+  }
+
+  img {
+    display: block;
+    max-height: 10em;
+    max-width: 100%;
+    margin: 0 auto;
+  }
+
+  table.ui {
+    margin-top: 1.5em;
+
+    td {
+      padding: 0.5em 1em;
+      line-height: 1.3em;
+
+      .rowLabel {
+        text-align: center;
+        font-weight: bold;
+      }
+
+      .exampleFunction {
+        background: #eee;
+        border-radius: 0.2em;
+        padding: 0.15em 0.3em;
+        font-family: "Courier New", Courier, monospace;
+        font-size: 0.9em;
+      }
+
+      .exampleInput {
+        background-color: #c9e8c0;
```

Comment:
Might be good to pull this into a var and share it with the styles for the text input distribution editor decorators too.